server/service/apple_mdm.go

```diff
@@ -6935,6 +6983,8 @@ func (svc *Service) MDMAppleProcessOTAEnrollment(
 		return nil, ctxerr.Wrap(ctx, err, "validating enroll secret")
 	}
 
+	// TODO(IB): Should we verify SSO check here, since this URL could be manually crafted and tampered.
```

Comment:
Given that we receive the cert, signer and all, I'm not sure how easily that could be tampered? I'd think validating the SSO check in the download OTA would be enough, but I'm not a security expert.

Reply:
You are right, verifying on getting the enrollment profile for ios and enrollment token link for android is enough.